### ISSUE TITLE
Getting wrong outputs and not/partly using KPU

### ISSUE THREAD

**Issue by MoritzKrauth** [kendryte/nncase#207]
Hey everyone. I made a very simple test with a Keras model which takes as input a 10x10x3 pixel image. Next I added a flatten and then a dense layer and wanted to distinguish 2 images with the model.
I trained the model and converted it to a uint8 kmodel. But now when I load this onto my k210 and pass the "kpu_run_kmodel" function these two images, the model cannot classify these two images. So the outputs of the "kpu_get_output" function are both always negative.

I created the kmodel with all three variants of the std/mean settings, as well as a float model. Always the same result.
What I also noticed: The "kpu_run_kmodel" function blocks the rest of the code from running. So the KPU is either not used at all or only partially. But there should be nothing in the model that does not meet the requirements.

class Model(Keras_Model):
    def __init__(self, dimension=(10,10,3)):
        super(Model, self).__init__()
        self.input_layer = tf.keras.layers.Input((10,10,3))
        self.flatten = Flatten()
        self.dense2 = Dense(2)
        self.out = self.call(self.input_layer)
        scce = tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True)
        self.compile(optimizer="adam", loss=scce, metrics=["accuracy"])

    def call(self, x):
        x = self.flatten(x)
        x = self.dense2(x)
        return x

I would appreciate a hint on how to fix these two problems.
If there is any information missing, I will be happy to provide it.
